index.bs

```diff
@@ -10522,6 +10526,9 @@ The [=remote end steps=] given |session| and |command parameters| are:
 1. Let |files| be the value of the |command parameters|["<code>files</code>"]
    field.
 
+1. If one of the |files| doesn't exist and the user agent can not handle such case,
```

Comment:
I think we should phrase this more like 

```
1. If, for any reason, the remote end is unable the [=selected files=] of |element| to the files with paths given in |files|, return error with [=error code=] [=unsupported operation=].

   Note: For example remote ends may be unable to set [=selected paths=] to files that do not currently exist on the filesystem.
````

We should also move it below the cancel steps, although I guess that will become moot once we change how cancellation works.

Reply:
Ok, maybe then below the part where update the files? Because I'm not sure if it's fine to put it before the `Otherwise`. I've updated it like this now

Reply:
@jgraham would you mind having a 2nd look so that we can merge this PR if you are fine?